framework/test/mocha/unit/modules/chain/helpers/config.js

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+'use strict';
+
+const _ = require('lodash');
+const config = require('../../../../../../src/modules/chain/helpers/config');
+const {
+	ZSchema,
+} = require('../../../../../../src/controller/helpers/validator');
+
+describe('helpers/config', () => {
+	const env = _.cloneDeep(process.env);
+	const argv = _.cloneDeep(process.argv);
+	const packageJsonStub = {
+		version: '1.5.0',
+		lisk: {
+			protocolVersion: '1.0',
+		},
+	};
+
+	beforeEach(async () => {
+		process.argv = _.cloneDeep(argv);
+		process.env = _.cloneDeep(env);
+
+		ZSchema.getLastErrors = sinonSandbox.stub();
+		process.exit = sinonSandbox.stub();
+		console.error = sinonSandbox.stub();
+	});
+
+	afterEach(async () => {
+		sinonSandbox.restore();
+		process.env = env;
+	});
+
+	it('should return appConfig containing correct properties', async () => {
+		const appConfig = config(packageJsonStub, false);
```

Comment:
`packageJsonStub` is a simple JSON object it's not a stub. Naming is misleading.